### PR TITLE
Bwockner/change detection overlay

### PR DESCRIPTION
**Pull request by bwockner**
added change detection blending with rgb

### CODE REVIEW

File: src/vcSettings.h [hunk 1]
```diff
@@ -123,7 +123,16 @@ struct vcVisualizationSettings
   int minIntensity;
   int maxIntensity;
 
-  udFloat2 displacement;
+  struct
+  {
+    float x;
+    float y;
```
Comment (samsinsane) [inline src/vcSettings.h]: This should be `udFloat2` not individual components, these two variables need to be contiguous and using a `udFloat2` ensures that.

File: src/vcSettings.cpp [hunk 2]
```diff
@@ -221,6 +221,11 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     pSettings->visualization.minIntensity = data.Get("visualization.minIntensity").AsInt(0);
     pSettings->visualization.maxIntensity = data.Get("visualization.maxIntensity").AsInt(65535);
 
+    pSettings->visualization.displacement.error = data.Get("visualization.displacement.errColour").AsInt(0x00FF00FF | 128<<24);
+    pSettings->visualization.displacement.max = data.Get("visualization.displacement.maxColour").AsInt(0x00FF0000 | 128<<24);
+    pSettings->visualization.displacement.min = data.Get("visualization.displacement.minColour").AsInt(0x0000FF00 | 56<<24);
+    pSettings->visualization.displacement.mid = data.Get("visualization.displacement.midColour").AsInt(0x0000FFFF | 56<<24);
```
[the remark this answers is not in the document]
Comment (bwockner) [reply src/vcSettings.cpp]: Around 50% (128) alpha is a good level to still show details from rgb, any higher starts to make the original image a bit hard to see.

The 56 is a little arbitrary (I wanted around 25% but for some reason went a bit lower). I feel using decimal to set alpha is justifiable as the UI uses decimal for setting colour channels and I personally find the decimal representation more intuitive in this case. Is there any particular reason that hex is better for this purpose?

[the remark this answers is not in the document]
Comment (samsinsane) [reply src/vcSettings.cpp]: Colours are often represented in hex, especially in code, the UI representation is irrelevant here. Additionally, bitwise operations are much easier to read and understand when written in hex.

**Comment by OsiJr** [comment]
I'm fixing this locally and have merged with #405

**Pull request closed without merging**